WebApiBackend/WebApiBackend/Model/Chores.cs

```diff
@@ -1,11 +1,13 @@
 ﻿using System;
```

Comment:
I think it's better if this file/class is renamed to "Chore" (not plural) since it represents a single chore and is consistent with Payment, User etc.

Reply:
done